fury/tests/test_actors.py

```diff
@@ -1208,5 +1208,58 @@ def test_superquadric_actor(interactive=False):
     npt.assert_equal(res.colors_found, [True, True, True])
 
 
+def test_square_actor(interactive=False):
+    from fury import utils, primitive as fp
+    scene = window.Scene()
+    centers = np.array([[2, 0, 0], [0, 2, 0], [0, 0, 0]])
+    colors = np.array([[255, 0, 0], [0, 255, 0], [0, 0, 255]])
+    scale = [1, 2, 3]
+
+    verts, faces = fp.prim_square()
+    res = fp.repeat_primitive(verts, faces, centers=centers, colors=colors,
+                              scale=scale)
+
+    big_verts, big_faces, big_colors, _ = res
+    sq_actor = utils.get_actor_from_primitive(big_verts, big_faces, big_colors)
+    sq_actor.GetProperty().BackfaceCullingOff()
+    scene.add(sq_actor)
+    scene.add(actor.axes())
+    if interactive:
+        window.show(scene)
+
+
+def test_canvas_actor(interactive=False):
+    scene = window.Scene()
+    scene.background((1, 1, 1))
+    centers = np.array([[2, 0, 0], [0, 2, 0], [0, 0, 0]])
+    colors = np.array([[255, 0, 0], [0, 255, 0], [0, 0, 255]])
+    scale = [1, 2, 1]
+
+    fake_sphere = \
+    """
+    float len = length(point);
+    float radius = 1.;
+    if(len > radius)
+        {discard;}
+
+    vec3 normalizedPoint = normalize(vec3(point.xy, sqrt(1. - len)));
+    vec3 direction = normalize(vec3(1., 1., 1.));
+    float df = max(0, dot(direction, normalizedPoint));
+    float sf = pow(df, 24);
+    fragOutput0 = vec4(max(df * color, sf * vec3(1)), 1);
+    """
+
+    canva_actor = actor.canvas(centers,
+                               colors=colors.astype(np.uint8),
+                               scale=scale,
+                               fs_impl=fake_sphere)
+    scene.add(canva_actor)
```

Comment:
Change to `canvas_actor`

Reply:
Native English-speaker here. Can confirm "canvas" is singular and "canvases" is plural.

Reply:
typo fixed!